src/common/components/vanilla/button/index.js

```diff
@@ -10,7 +10,7 @@ const defaultProps = {
   onClick: PropTypes.func,
   type: PropTypes.string,
   appearance: React.PropTypes.oneOf(['primary', 'secondary', 'positive', 'negative', 'neutral']),
-  flavour: React.PropTypes.oneOf(['normal','embiggened']),
+  flavour: React.PropTypes.oneOf(['normal','embiggened', 'ensmallened']),
```

Comment:
Haha, I forgot I wanted to rename these into `bigger` and `smaller`, as this is going into a crazy direction :)